src/app/components/Modal/FeedbackModal.tsx

```diff
@@ -0,0 +1,187 @@
+"use client";
+
+import { useState } from "react";
+
+type ModalProps = {
+	isVisible: boolean;
+	onClose: () => void;
+	pageTitle: string;
+};
+
+const Modal = ({ isVisible, onClose, pageTitle }: ModalProps) => {
+	const [isQuestionOneDropdown, setIsQuestionOneDropDown] = useState(false);
+	const [isQuestionTwoDropdown, setIsQuestionTwoDropDown] = useState(false);
+
+	const toggleQuestionOneDropdown = () => {
+		setIsQuestionOneDropDown(!isQuestionOneDropdown);
+		if (isQuestionTwoDropdown) {
+			setIsQuestionTwoDropDown(false);
+		}
+	};
+	const toggleQuestionTwoDropdown = () => {
+		setIsQuestionTwoDropDown(!isQuestionTwoDropdown);
+		if (isQuestionOneDropdown) {
+			setIsQuestionOneDropDown(false);
+		}
+	};
+
+	if (!isVisible) return null;
+
+	return (
+		<div className="fixed inset-0 bg-black bg-opacity-50 backdrop-blur-sm flex justify-center items-center">
+			<div className="w-[55%] h-[61%] flex flex-col p-8 rounded-lg shadow bg-[#f9f7fc]">
```

Comment:
we should never set height unless it is a small item. Could you use padding plaease?

Reply:
resolved